templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -3,254 +3,346 @@ mode: 0644
-    seccomp_profile = "/etc/crio/seccomp.json"
+    # A list of ulimits to be set in containers by default, specified as
+    # "<ulimit name>=<soft limit>:<hard limit>", for example:
+    # "nofile=1024:2048"
+    # If nothing is set here, settings will be inherited from the CRI-O daemon
+    #default_ulimits = [
+    #]
 
-    # apparmor_profile is the apparmor profile name which is used as the
-    # default for the runtime.
-    apparmor_profile = "crio-default"
+    # default_runtime is the _name_ of the OCI runtime to be used as the default.
+    # The name is matched against the runtimes map below.
+    # default_runtime = "runc"
 
-    # cgroup_manager is the cgroup management implementation to be used
-    # for the runtime.
-    cgroup_manager = "systemd"
+    # If true, the runtime will not use pivot_root, but instead use MS_MOVE.
+    # no_pivot = false
 
-    # default_capabilities is the list of capabilities to add and can be modified here.
-    # If capabilities below is commented out, the default list of capabilities defined in the
-    # spec will be added.
-    # If capabilities is empty below, only the capabilities defined in the container json
-    # file by the user/kube will be added.
-    default_capabilities = [
-      "CHOWN", 
-      "DAC_OVERRIDE", 
-      "FSETID", 
-      "FOWNER", 
-      "NET_RAW", 
-      "SETGID", 
-      "SETUID", 
-      "SETPCAP", 
-      "NET_BIND_SERVICE", 
-      "SYS_CHROOT", 
-      "KILL", 
-    ]
+    # Path to the conmon binary, used for monitoring the OCI runtime.
+    # Will be searched for using $PATH if empty.
+    conmon = "/usr/libexec/crio/conmon"
 
-    # hooks_dir_path is the oci hooks directory for automatically executed hooks
-    hooks_dir_path = "/usr/share/containers/oci/hooks.d"
+    # Cgroup setting for conmon
+    # conmon_cgroup = "pod"
 
-    # default_mounts is the mounts list to be mounted for the container when created
-    # deprecated, will be taken out in future versions, add default mounts to either
-    # /usr/share/containers/mounts.conf or /etc/containers/mounts.conf
-    default_mounts = [
-      "/usr/share/rhel/secrets:/run/secrets", 
-    ]
+    # Environment variable list for the conmon process, used for passing necessary
+    # environment variables to conmon or the runtime.
+    # conmon_env = [
+    # 	"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+    # ]
 
-    # Path to directory in which container exit files are written to by conmon.
-    container_exits_dir = "/var/run/crio/exits"
+    # If true, SELinux will be used for pod separation on the host.
+    # selinux = true
 
-    # Path to directory for container attach sockets.
-    container_attach_socket_dir = "/var/run/crio"
+    # Path to the seccomp.json profile which is used as the default seccomp profile
+    # for the runtime. If not specified, then the internal default seccomp profile
+    # will be used.
+    seccomp_profile = "/etc/crio/seccomp.json"
```

Comment:
And this?

Reply:
ah actually I don't think this one is needed. good catch

Reply:
fixed